mod-source-record-manager-server/src/main/java/org/folio/services/journal/JournalUtil.java

```diff
@@ -201,18 +203,18 @@ private static JournalRecord buildJournalRecordWithMarcBibType(JournalRecord.Act
       else actionTypeForMarcBib = JournalRecord.ActionType.UPDATE;
     }
 
-    return buildCommonJournalRecord(actionStatus, actionTypeForMarcBib, currentRecord, eventPayload, eventPayloadContext)
+    return buildCommonJournalRecord(actionStatus, actionTypeForMarcBib, currentRecord, eventPayload, eventPayloadContext, incomingRecordId)
       .withEntityId(marcBibEntityId)
       .withEntityType(MARC_BIBLIOGRAPHIC);
   }
 
   private static JournalRecord buildCommonJournalRecord(JournalRecord.ActionStatus actionStatus, JournalRecord.ActionType actionType, Record currentRecord,
-                                                        DataImportEventPayload eventPayload, HashMap<String, String> eventPayloadContext) {
+                                                        DataImportEventPayload eventPayload, HashMap<String, String> eventPayloadContext, String incomingRecordId) {
     String tenantId = eventPayload.getContext().get(CENTRAL_TENANT_ID_KEY);
 
     var currentJournalRecord = new JournalRecord()
       .withJobExecutionId(currentRecord.getSnapshotId())
-      .withSourceId(currentRecord.getId())
+      .withSourceId(incomingRecordId)
```

Comment:
looks like we need also to update the "get_job_log_entries" and "get_record_processing_log" functions so that they will return recordId from the "entity_id" column instead of "source_id" since now "source_id" column is being populated with incomingRecordId. Otherwise, suppose, ui will not get a record from srs to display on the log page.